apps/spruce/src/pages/waterfall/WaterfallGrid.tsx

```diff
@@ -21,9 +23,26 @@ type WaterfallGridProps = {
   projectIdentifier: string;
 };
 
+const tracer = trace.getTracer(TRACER_NAME);
```

Comment:
I wonder if we want to use a different tracer name here (you can use any name you want I believe, like `performance`, `render`, etc)

Calling it with `analytics` tracer name means it shows up in our events like this:

<img width="427" alt="Screenshot 2024-11-01 at 10 48 26 AM" src="https://github.com/user-attachments/assets/5ffe07fa-f8c5-42a6-877a-0a00edf9ee50">

or we can rename the event to follow our conventions as `Rendered waterfall` and add it to the analytics spreadsheet